website/docs/cdktf/concepts/stacks.mdx

```diff
@@ -190,7 +190,7 @@ If you e.g. write `this.allResources = Fn.merge(resourceFromStackA.items, resour
 In case you don't want the "middleman" to be cut out you can run `stackD.addDependency(stackC)` to make the dependency explicit.
 If you want to keep the result of the function attached to one stack (and saved its state) you can use a `TerraformLocal` and expose it instead: `this.allResources = TerraformLocal(this, "merged_items", Fn.merge(resourceFromStackA.items, resourceFromStackB.items)`.
```

Comment:
```suggestion
To make the dependency explicit, run`stackD.addDependency(stackC)`.

If you want to keep the result of the function attached to one stack and save its state, create a [Terraform Local value](/cdktf/concepts/variables-and-outputs#local-values) and expose it. The following example creates a `TerraformLocal`.

```ts
this.allResources = TerraformLocal(this, "merged_items", Fn.merge(resourceFromStackA.items, resourceFromStackB.items)
```
```